nncf/torch/quantization/metrics.py

```diff
@@ -348,16 +350,13 @@ def _marking_edges(self, graph, node_key, queue, mark=True):
                 self.stats.quantized_edges_in_cfg += 1
 
     def get_merged_original_graph_with_patterns(self, original_graph: PTNNCFGraph):
-        import nncf.torch.graph.patterns as p
-        from nncf.common.graph.graph_matching import find_subgraphs_matching_expression
-
-        pattern = p.LINEAR_OPS + p.ANY_BN_ACT_COMBO | p.LINEAR_OPS + p.ELTWISE_UNIFORM_OPS
+        pattern = get_full_pattern_graph()
         # pylint: disable=protected-access
         matches = find_subgraphs_matching_expression(original_graph._nx_graph, pattern)
         merged_graph = deepcopy(original_graph._nx_graph)
         nx.set_node_attributes(merged_graph, False, self.IS_MERGED_GRAPH_ATTR)
         for match in matches:
-            if len(match) == 1:
+            if len(match) <= 1:
```

Comment:
Empty matches should not be returned at all

Reply:
As return value is [[]] we could have an empty match and this is the case why we need `<=`

Reply:
Using `[]` as return value will solve this.

Reply:
Done